media-sound/qastools/qastools-0.23.0.ebuild

```diff
@@ -7,12 +7,12 @@ inherit cmake
 
 DESCRIPTION="Qt5 GUI ALSA tools: mixer, configuration browser"
 HOMEPAGE="https://gitlab.com/sebholt/qastools"
-SRC_URI="https://gitlab.com/sebholt/${PN}/-/archive/v${PV}/${PN}-v${PV}.tar.gz"
+SRC_URI="https://gitlab.com/sebholt/${PN}/-/archive/v${PV}/${PN}-v${PV}.tar.bz2"
```

Comment:
`.tar.gz` -> `.tar.bz2` without updating Manifest? This is probably going to blow up. Why even switch the tarball type?

Reply:
But, and **again**, you should run **pkgcheck scan** on your **own** changes before pushing if you're fixing **pkgcheck related** warnings to make sure it passes...

I and we all do appreciate it, but you can improve your workflow to make it easier!